plugins/base/src/main/resources/edu/wpi/first/shuffleboard/plugin/base/widget/RelayWidget.fxml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ToggleButton?>
+<?import javafx.scene.control.ToggleGroup?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<StackPane fx:id="root" stylesheets="@relay-widget.css" xmlns="http://javafx.com/javafx/8.0.121"
+           xmlns:fx="http://javafx.com/fxml/1"
+           fx:controller="edu.wpi.first.shuffleboard.plugin.base.widget.RelayWidget">
+    <VBox>
+        <children>
+            <ToggleButton id="offButton" fx:id="offButton" alignment="CENTER" maxHeight="Infinity" maxWidth="Infinity"
+                          mnemonicParsing="false" text="Off">
+                <toggleGroup>
+                    <ToggleGroup fx:id="state"/>
+                </toggleGroup>
+            </ToggleButton>
+            <ToggleButton id="onButton" fx:id="onButton" maxHeight="Infinity" maxWidth="Infinity"
+                          mnemonicParsing="false" text="On" toggleGroup="$state"/>
+            <ToggleButton id="forwardButton" fx:id="forwardButton" maxHeight="Infinity" maxWidth="Infinity"
+                          mnemonicParsing="false" text="Forward" toggleGroup="$state"/>
+            <ToggleButton id="reverseButton" fx:id="reverseButton" maxHeight="Infinity" maxWidth="Infinity"
```

Comment:
These buttons don't grow vertically, which results in a lot of empty space at large heights

![relay-buttons-don t-grow](https://user-images.githubusercontent.com/6320992/32137135-03c48af2-bbe8-11e7-95a4-2a7e74b69acb.PNG)

Reply:
It'd also be nice to have some spacing between buttons... `2` or maybe `4` would work

Reply:
@sam if all that's left to fix is the buttons growing vertically, which is not clearly the right thing to do, then is it OK to merge now?